### ISSUE TITLE
I am having issues getting the project to run Issue with sparkle

### ISSUE THREAD

**Issue by elmcapp** [krevis/MIDIApps#102]
I am having issues getting the project to run. please tell me if I am missing something.

1 open Mac terminal and cd into the Downloads directory
2. git clone https://github.com/krevis/MIDIApps.git
3. cd MIDIApps
4. in terminal I enter the follow: git submodule update --init --recursive
5. Open MIDIApps.xcworkspace in Xcode
6. I open and edit Configurations/Snoize-Signing.xcconfig.  with my team id
7. In the "Scheme" popup menu  I select MidiMonitor and then I press the run button

Errors

![Screen Shot 2022-03-22 at 5 41 38 PM](https://user-images.githubusercontent.com/64275658/159580981-936cc12f-6256-42f3-b5a0-1c15cb18b778.png)

**Comment by krevis** [comment]
This looks like a bug with Xcode or the Swift package manager. 

Here's a similar case with a different project: https://github.com/firebase/firebase-ios-sdk/issues/8812
That suggests you should try these steps:

1. Quit xCode.
2. `rm -rf ~/Library/Developer/Xcode/DerivedData/`
3. `rm -rf ~/Library/Caches/org.swift.swiftpm/`
4. Open xCode.
5. Reset Packages Caches: `File -> Packages -> Reset Package Caches`

I haven't seen this myself, I just googled for 'swift package "already exists in file system"'.

**Comment by elmcapp** [comment]
Worked like a charm. For my future reference. Could you please tell me how is it that you know where to start looking for this issue. What stood out to you to check or know what to google. 

@krevis I just reached out to you via your support email for a different issue

**Comment by krevis** [comment]
Well, I knew I recently changed Sparkle from being a git submodule to using Swift Package Manager, and the error message mentioned `Sparkle-For-Swift-Package-Manager`, so it seemed very likely to be a SPM problem.

I just googled the key words in the error message (the ones that were not specific to your paths or the Sparkle project).

The message also mentioned a file in your Xcode `DerivedData` directory, and it's a common Xcode troubleshooting step to remove your DerivedData and try again, so the advice in that other bug seemed plausible. (And harmless if it didn't work.)

It's generally up to Xcode and SPM to manage the package files in DerivedData, not something a build script in my project would do, so I didn't even bother looking at my project.
